src/mpid/ch4/src/ch4_probe.h

```diff
@@ -31,6 +32,7 @@ MPL_STATIC_INLINE_PREFIX int MPID_Probe(int source,
         goto fn_exit;
     }
 
+    av = MPIDIU_comm_rank_to_av(comm, source);
```

Comment:
Ah. I wasn't look at all of the patches at once when I made this comment. When looking at the patch where the above line was introduced, there was no use of `av` in the non-exclusive code.